workflows/prognostic_run_diags/fv3net/diagnostics/prognostic_run/metrics.py

```diff
@@ -238,8 +276,7 @@ def register_parser(subparsers):
 
 
 def main(args):
-    diags = xr.open_dataset(args.input)
-    diags["time"] = diags.time - diags.time[0]
```

Comment:
none of the metrics actually depend on the time coordinate starting at zero, so I'm removing this.